package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "3.9.10",
+  "version": "3.9.11",
```

Comment:
I want to merge this in before the date PR.

Reply:
... and before the sbc-common-components PR.

(Because this ticket is low risk and doesn't need much testing, whereas the other 2 tickets need actual QA.)